src/quo/components/avatars/user_avatar/schema.cljs

```diff
@@ -0,0 +1,26 @@
+(ns quo.components.avatars.user-avatar.schema
+  (:require
+    [quo.components.avatars.user-avatar.style :as style]))
+
+
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map
+      [:full-name {:optional true} [:maybe string?]]
+      [:size {:optional true} [:maybe (into [:enum] (keys style/sizes))]]
+      [:customization-color [:maybe :schema.common/customization-color]]
+      [:static? {:optional true} [:maybe :any]]
+      [:status-indicator? {:optional true} [:maybe :any]]
+      [:online? {:optional true} [:maybe :any]]
+      [:ring? {:optional true} [:maybe :any]]
```

Comment:
would these be booleans given the boolean-sounding names, or are they used differently?